### ISSUE TITLE
[CoE Starter Kit - BUG] Set App Permissions Getting failure on Helper Object Operations when adding permissions to a person

### ISSUE THREAD

**Issue by mdphung** [microsoft/coe-starter-kit#7147]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

Came back in and logged out and came back in and tried the set app permissions app again on another app that was orphaned and added a new owner.

Got this error message:
![image](https://github.com/microsoft/coe-starter-kit/assets/117682988/fa1f5d92-2d6d-48ef-b2fe-2729e70254ed)

This is what is happening in the flow, this error:

![image](https://github.com/microsoft/coe-starter-kit/assets/117682988/26d6b27a-2720-4f30-9b76-3986de0d3117)

Is there something else that can be done.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.13

### What app or flow are you having the issue with?

HELPER - ObjectOperations

### What method are you using to get inventory and telemetry?

Cloud flows

**Comment by Jenefer-Monroe** [comment]
Yes it looks like we forgot to update this old app when we added group ability in the PPAdmin View version. 
We are replacing these apps but it might not be ready for Dec release so I'll fix this up real quick for that release.

**Comment by Jenefer-Monroe** [comment]
ok, actually it seems like this has been resolved in more recent builds. You show as being on August release, and in current releases we do manage this as an optional variable
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/269fc212-0172-42a6-af20-e9e7dcf10aaf)

**Comment by mdphung** [comment]
@Jenefer-Monroe I am actually in 4.17 so I am in the November Build my apologies for putting the wrong build number

**Comment by Jenefer-Monroe** [comment]
Can you see if you have unmanaged layers on these flows? This has been been addressed so hoping you can just get unblocked that way.

- HELPER - CloudFlowOperations
- HELPER - CanvasAppOperations
- HELPER - ObjectOperations

**Comment by mdphung** [comment]
None of these flows are unmanaged layers.

**Comment by Jenefer-Monroe** [comment]
Maybe we're just looking at the wrong error message. 
This should fail for you and be caught. Can you please validate that? 
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/863472ac-2597-42aa-a1d2-10613c4f62bc)

I believe this action is failing for you, is that correct? What error message do you get there?
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/ec45703a-2261-49df-918c-57ed06e613b0)

**Comment by Jenefer-Monroe** [comment]
closing out as no further action for starter kit team
